recipes/avahi/all/conanfile.py

```diff
@@ -98,9 +99,9 @@ def package(self):
     def package_info(self):
         for lib in ("client", "common", "core", "glib", "gobject", "libevent", "compat-libdns_sd"):
             avahi_lib = f"avahi-{lib}"
+            self.cpp_info.components[lib].set_property("pkg_config_name", avahi_lib)
             self.cpp_info.components[lib].names["cmake_find_package"] = lib
             self.cpp_info.components[lib].names["cmake_find_package_multi"] = lib
-            self.cpp_info.components[lib].names["pkg_config"] = avahi_lib
```

Comment:
Will removing this break Conan V1 packages that use the pkg-config files?

Reply:
Oh wait, I think it will only effect the old Conan V1 `PkgConfig` generator and not the newer `PkgConfigDeps` generator. I'm fine with removing support for that. Thanks again for the contribution!